### PR TITLE
[Update] AnimeFever: fix for new search page, removed old code

### PR DESCRIPTION
**Pull request by Venipa**
Browsing Default:
image
View Anime:
image
Viewing Episode:
Paused:
image
Play:
image
Schedule and other pages:
image
(New) Search Page (was broken before due to new ui):
image
image

### CODE REVIEW

File: websites/A/AnimeFever/presence.ts [hunk 1]
```diff
@@ -64,30 +53,30 @@ function getQuery() {
...
+            if (!entry) return l;
             return Object.assign(l, { [entry[0]]: entry[1] });
           }, {})
         : {};
   return query;
 }
 function getAnimeEntity(): AnimeEntity {
   const object = Array.from(
-    document.querySelectorAll(`script[type="application/ld+json"]`)
+    document.querySelectorAll("script[type=\"application/ld+json\"]")
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:64]: escaping. 
```suggestion
    document.querySelectorAll('script[type="application/ld+json"]')
```

File: websites/A/AnimeFever/presence.ts [hunk 2]
```diff
@@ -64,30 +53,30 @@ function getQuery() {
...
   ).find(
     (x) =>
       x.textContent.indexOf(location.pathname) !== -1 &&
-      x.textContent.indexOf(`"@type":"TVSeries"`) !== -1
+      x.textContent.indexOf("\"@type\":\"TVSeries\"") !== -1
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:68]:
```suggestion
      x.textContent.indexOf('"@type":"TVSeries"') !== -1
```

File: websites/A/AnimeFever/presence.ts [hunk 3]
```diff
@@ -64,30 +53,30 @@ function getQuery() {
...
   )?.textContent;
   if (!object) return null;
   return JSON.parse(object);
 }
 function getAnimeEpsiodeEntity(): AnimeEpisodeEntity {
   const object = Array.from(
-    document.querySelectorAll(`script[type="application/ld+json"]`)
+    document.querySelectorAll("script[type=\"application/ld+json\"]")
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:75]:
```suggestion
    document.querySelectorAll('script[type="application/ld+json"]')
```

File: websites/A/AnimeFever/presence.ts [hunk 4]
```diff
@@ -64,30 +53,30 @@ function getQuery() {
...
   ).find(
     (x) =>
       x.textContent.indexOf(location.pathname) !== -1 &&
-      x.textContent.indexOf(`"@type":"TVEpisode"`) !== -1
+      x.textContent.indexOf("\"@type\":\"TVEpisode\"") !== -1
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:79]:
```suggestion
      x.textContent.indexOf('"@type":"TVEpisode"') !== -1
```

File: websites/A/AnimeFever/presence.ts [hunk 5]
```diff
@@ -122,8 +112,8 @@ function getAnimeEpsiodeEntity(): AnimeEpisodeEntity {
             if (data.startTimestamp) delete data.startTimestamp;
             if (data.endTimestamp) delete data.endTimestamp;
           } else {
-            data.startTimestamp = videoTime[0];
-            data.endTimestamp = videoTime[1];
+            data.startTimestamp = startTimestamp;
+            data.endTimestamp = endTimestamp;
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:116]: you don't need startTimestamp :P

File: websites/A/AnimeFever/presence.ts [hunk 6]
```diff
@@ -146,46 +136,14 @@ function getAnimeEpsiodeEntity(): AnimeEpisodeEntity {
...
-                .slice(0, 5)
-                .join(", ")}`;
-            } else {
-              data.state = "-";
-            }
-          } else {
-            data.state = searchedValue;
-          }
+          const searchedValue = document.querySelector<HTMLInputElement>(
+            ".shows-content input[type=\"text\"]"
```
Comment (ririxidev) [inline websites/A/AnimeFever/presence.ts:140]:
```suggestion
            '.shows-content input[type="text"]'
```

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**